### ISSUE TITLE
why wrap component with `div`?

### ISSUE THREAD

**Issue by kentor** [chenglou/react-radio-group#18]
The render method for the radio group returns a div:

```javascript
render: function() {
  let {name, selectedValue, onChange, children} = this.props;
  return (
    <div>
      {children && children(radio(name, selectedValue, onChange))}
    </div>
  );
}
```

But if the `children` function can only return one value, which is assumed to be an element, then why the need to wrap everything in a `<div>`?

I think it can just return calling the children:

```javascript
render: function() {
  let {name, selectedValue, onChange, children} = this.props;
  return children ? children(radio(name, selectedValue, onChange)) : null;
}
```

**Comment by chenglou** [comment]
The children function can actually return e.g. an array of values, thus the wrapper.

But I should probably follow https://github.com/chenglou/react-motion/issues/44

**Comment by chenglou** [comment]
Released as 2.1.0.

**Comment by kentor** [comment]
Great, thanks. :+1:
